.github/ISSUE_TEMPLATE/bugreport.yml

```
@@ -0,0 +1,102 @@
+name: Bug Report
+description: Report a bug building or running ungoogled-chromium
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before submitting this issue, please confirm if you should submit it to a [platform-specific issue tracker instead](//github.com/Eloston/ungoogled-chromium/blob/master/SUPPORT.md#platform-specific-support)
+
+        Otherwise, your issue may not be noticed. In addition, please read through the [SUPPORT.md](//github.com/Eloston/ungoogled-chromium/blob/master/SUPPORT.md) first
+  - type: dropdown
+    id: os
+    attributes:
+      label: OS/Platform
+      description: OS/Platform you are running ungoogled-chromium on
+      options:
+          - Android
+          - Arch Linux
+          - Debian, Ubuntu, and derivatives
+          - Fedora and CentOS
+          - Gentoo
+          - GNU Guix
+          - macOS
+          - NixOS
+          - Portable Linux
+          - Windows
+    validations:
+        required: true
+  - type: dropdown
+    id: install
+    attributes:
+      label: Installed
+      description: How was ungoogled-chromium installed on your OS/Platform
+      options:
+          - Compiled from source
+          - OS/Platform's package manager
+          - Flatpak
+          - https://ungoogled-software.github.io/ungoogled-chromium-binaries/
+          - Other (please use 'Additional context' field below to mention how exactly)
+    validations:
+        required: true
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: ungoogled-chromium version. Could be copied from chrome://settings/help
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: Tested vanilla Chromium?
+      options:
+        - label: I have tried reproducing this issue in vanilla Chromium and it could not be reproduced there
+  - type: checkboxes
+    attributes:
+      label: Tested Chrome?
+      options:
+        - label: I have tried reproducing this issue in Chrome and it could not be reproduced there
```

This should probably be one set of checkboxes; IMO it looks a bit cleaner.
```suggestion
  - type: checkboxes
    attributes:
      label: Tested upstream?
      options:
        - label: I have tried reproducing this issue in Chrome and it could not be reproduced there
        - label: I have tried reproducing this issue in vanilla Chromium and it could not be reproduced there
```

---

Reasonable.